addons/event/models/event_registration.py

```diff
@@ -167,7 +168,8 @@ def write(self, vals):
 
         ret = super(EventRegistration, self).write(vals)
 
-        if vals.get('state') == 'open':
+        if vals.get('state') == 'open' and not self.env.context.get('install_mode', False):
+            # avoid running scheduler for demo data
```

Comment:
Maybe tell why we don't want to run scheduler, not what we are actually doing :)